packages/core/core/src/AssetGraph.js

```diff
@@ -220,6 +226,96 @@ export default class AssetGraph extends Graph {
     }
   }
 
+  getDependencies(asset: Asset): Array<Dependency> {
```

Comment:
How do we feel about adding all of these helper methods meant for external plugin consumption to the existing AssetGraph class used in core?

Reply:
I'd rather not expose the raw AssetGraph. Ideally we'd wrap it and only expose what we want the plugins to use.

Reply:
Yeah I agree with that. Let's get this PR in first and then discuss what the interface we want to expose is.